### PR TITLE
#178

### PR DESCRIPTION
**Pull request by cmk-jarek**
Add option to include object not bound to a Namespace, Defaults change to output only the bound ones
refers to #178

### CODE REVIEW

File: pkg/kor/kor.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type IncludeExcludeLists struct {
 type Opts struct {
 	DeleteFlag    bool
 	NoInteractive bool
+	NoNamespaced  bool
```
Comment (yonahd) [inline pkg/kor/kor.go:35]: Rename to nonNamespaced

File: pkg/kor/all.go [hunk 2]
```diff
@@ -201,25 +201,27 @@ func GetUnusedAll(includeExcludeLists IncludeExcludeLists, filterOpts *FilterOpt
...
-	outputBuffer.WriteString(pvOutput)
-	outputBuffer.WriteString("\n")
-	noNamespaceResourceMap[pvDiff.resourceType] = pvDiff.diff
-
-	output := FormatOutputAll("", allDiffs, opts)
-
-	outputBuffer.WriteString(output)
-	outputBuffer.WriteString("\n")
-	response[""] = noNamespaceResourceMap
+	if opts.NoNamespaced {
```
Comment (yonahd) [inline pkg/kor/all.go:204]: This should be true in a case that a person chooses all namespaces(or no namespace which is all)
The Readme should reflect this

**Comment by yonahd** [comment]
@cmk-jarek Do you intend to complete this fix?

**Comment by cmk-jarek** [comment]
Sorry was on vacation will fix that this week as soon as i catch up.

**Comment by yonahd** [comment]
> Sorry was on vacation will fix that this week as soon as i catch up.

@cmk-jarek any update here?
I think you were pretty much done

**Comment by cmk-jarek** [comment]
sorry got burried. will try to fix that by the weekend.

**Comment by yonahd** [comment]
> sorry got burried. will try to fix that by the weekend.

Any chance this can be done this week?
I'd like to have the fix for the upcoming release

**Pull request closed without merging**